htdocs/core/lib/functions.lib.php

```diff
@@ -1552,6 +1554,11 @@ function dol_banner_tab($object, $paramid, $morehtml='', $shownav=1, $fieldid='r
 		$morehtmlref.='</div>';
 	}
 
+	$parameters = array();
```

Comment:
Can you move this new hook lower, inside the function showrefnav, jsut after hook moreHtmlStatus